src/test/java/seedu/mindmymoney/command/ListCommandTest.java

```diff
@@ -0,0 +1,28 @@
+package seedu.mindmymoney.command;
+
+import org.junit.jupiter.api.Test;
+import seedu.mindmymoney.MindMyMoneyException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class ListCommandTest {
+    @Test
+    void listToString_happyPath_expectString() {
+        String inputString = "expenditure 12345";
+        new AddCommand(inputString).executeCommand();
+        String listInString = new ListCommand().listToString();
+        assertEquals("1. $12345 on expenditure\n", listInString);
+        String inputString2 = "expenditure2 54321";
```

Comment:
Maybe add a new line before this line? To "break up" the 2 test cases.
```suggestion

        String inputString2 = "expenditure2 54321";
```